src/user-service/utils/validator.js

```diff
@@ -0,0 +1,11 @@
+const pool = require("../db.js");
+
+exports.checkIfNameOrEmailExists = async (username, email) => {
+    const user = await pool.query("SELECT * FROM Users WHERE username = $1 OR email = $2", [username, email]);
+    
+    if (user.rows.length > 0) {
```

Comment:
```suggestion
    return user.rows.length > 0;
```
This might be cleaner